build/.htaccess

```diff
@@ -0,0 +1 @@
+Deny from all
```

Comment:
/tests already have an .htaccess I just copied the behaviour.

I added the commented rules in the case someone preferred not to use the .htaccess (which is faster I think)